Mono.Cecil/AssemblyWriter.cs

```diff
@@ -104,8 +104,8 @@ static void Write (ModuleDefinition module, Disposable<Stream> stream, WriterPar
 				symbol_writer_provider = new DefaultSymbolWriterProvider ();
 
 #if !NET_CORE
-			if (parameters.StrongNameKeyPair != null && name != null) {
-				name.PublicKey = parameters.StrongNameKeyPair.PublicKey;
+			if (parameters.HasStrongNameKey && name != null) {
```

Comment:
Shouldnt the 

    #if !NET_CORE

Be removed?

Reply:
Possibly yes, but this change is orthogonal to the upgrade to .NET Standard 2.0, so I'd left it in.